easybuild/easyconfigs/t/tmux/tmux-1.9a-GCC-4.9.2.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'tmux'
+version = '1.9a'
+
+homepage = 'http://tmux.sourceforge.net/'
+description = """tmux is a terminal multiplexer. It lets you switch easily 
+between several programs in one terminal, detach them (they keep running in the background) and reattach them to a different terminal."""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+toolchainopts = {'optarch': True}
+
+#http://sourceforge.net/projects/tmux/files/tmux/tmux-1.9/tmux-1.9a.tar.gz/download
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://downloads.sourceforge.net/%(name)s/']
```

Comment:
@boegel Are these listed/documented somewhere?  More and more pop up in reviiews...